CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
+## Unreleased - as of Sprint 53 end 2017-01-30
+
+### Added
+- Refactor endpoint [(#64)](https://github.com/ansible/ansible_tower_client_ruby/pull/64)
+- Expose playbooks off of projects [(#62)](https://github.com/ansible/ansible_tower_client_ruby/pull/62)
+- Add update delete [(#61)](https://github.com/ansible/ansible_tower_client_ruby/pull/61)
```

Comment:
Add `update` and `delete` methods to all exposed classes.

@syncrou Sound right?